Plots/Skew-T/NCL_skewt_3_2.py

```diff
@@ -0,0 +1,117 @@
+"""
+NCL_Skew_3_2.py
+==================
+This script illustrates the following concepts:
+    - Drawing Skew-T plots
+    - Thinning the wind barbs in a Skew-T plot
+    - Customizing the background of a Skew_T plot
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/skewt_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/skewt_3_2_lg.png
+"""
+###############################################################################
+# Import packages:
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.lines as mlines
+import pandas as pd
+import metpy.calc as mpcalc
+from metpy.plots import  SkewT
+from metpy.units import units
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = pd.read_csv(gdf.get('ascii_files/sounding_ATS.csv'),  header=None)
+
+# Extract the data
+p = ds[0].values*units.hPa   # Pressure [mb/hPa]
+tc = ds[1].values*units.degC  # Temperature [C]
+tdc = ds[2].values*units.degC  # Dew pt temp  [C]
+z = ds[4]   # Geopotential [gpm]
```

Comment:
Geopotential height isn't used anywhere and doesn't need to be extracted from the data.